internal/samples/ansible/constants.go

```diff
@@ -229,12 +229,6 @@ const taskForSecret = `- name: Create test service
           port: 8332
           targetPort: 8332
           name: rpc
-
-- name: Check if jmespath is installed
-  set_fact:
-    instance_tags: '{{app | json_query(query)}}'
-  vars:
-    query: 'app[*]."memcached"'
```

Comment:
since it is an invalid code and has not been used at all we agreed in to remove it.
also, it should not be in the secret and we raise an issue to see if we should check if the deps are installed. See: https://github.com/operator-framework/operator-sdk/issues/4123